deploy_stable.bat

```diff
@@ -0,0 +1,2 @@
+git switch main
```

Comment:
This won't work if there are local changes

Reply:
Correct. `git stash` the changes as well.

Reply:
Tbh I'd advocate for git reset --hard and some sort of reset to TOT then a switch.

Reply:
I think we should err on the side of caution. We can always clear the stash, but resetting something potentially left on the laptop on accident is not recoverable.

Reply:
Changes shouldn't exist on the drive laptop

Reply:
at competitions we might be working on a competition sandbox branch

Reply:
Not on the drive laptop you shouldn't be

Reply:
Alright I think I'll agree with that. Use `git reset --hard HEAD`.

Reply:
Use `git switch -f main`